infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api-no-calculated-variables.bicep

```diff
@@ -1,6 +1,6 @@
-// Note that we don't set the connection strings when creating the tfm-api webapp.
+// Note that we don't set the calculated variables & connection strings when creating the tfm-api webapp.
 // This is because we need to include the TFM front door hostname which would set
-// up a circular dependency. We avoid this by setting the connection strings
+// up a circular dependency. We avoid this by setting the calculated variables
 // separately later.
```

Comment:
Again, that's not quite what's going on. You replace all of the variables with the combination of these values and the calculated ones later. You do _not_ add the calculated ones to the existing set.